aodntools/timeseries_products/common.py

```diff
@@ -128,4 +124,34 @@ def check_file(nc, site_code, variables_of_interest):
     # check qc flag conventions for VoI and depth/pressure
     error_list.extend(check_imos_flag_conventions(nc))
 
-    return error_list
\ No newline at end of file
+    return error_list
+
+
+def check_velocity_file(nc, site_code,
+                        required_variables=('TIME', 'DEPTH', 'LATITUDE', 'LONGITUDE', 'UCUR', 'VCUR'),
```

Comment:
WCUR missing?

Reply:
I don't know if it is because I'm using lambdas heavily lately, but this could be a lambda if it's just a wrapper. This would avoid  that monster of docstring (since most of it is already/should be within check_file fun).

Reply:
> WCUR missing?

No, WCUR is not required. Although it _is_ actually there in just about every file, we explicitly said in the plan and the [documentation](https://github.com/aodn/python-aodntools/blob/master/aodntools/timeseries_products/Documentation/velocity_aggregated_timeseries.md#input) that only UCUR and VCUR are absolutely required. I'm no longer sure why we did this, but that was the decision. Eduardo added [code](https://github.com/aodn/python-aodntools/blob/b68c421ce1258460dabaa7b6cc9c48cdfc0de1bb/aodntools/timeseries_products/velocity_aggregated_timeseries.py#L205) to handle the case where WCUR is missing.